content/en/Integrations/IntegrationBuilder/configure-gitlab-integration.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Configure GitLab Integration"
+linkTitle: "Configure GitLab Integration"
+weight: 40
+---
+
+## Configure GitLab Integration for New Findings
+
+To configure GitLab to create issues for new findings, you need three general steps:
+
+1. Create a Cobalt connection.
+2. Obtain a GitLab project access token.
+3. Create a GitLab connection on the Cobalt platform.
+4. Create a new recipe on the Cobalt platform.
+
+### 1. Create a Cobalt Connection.
+
+Find out how to create a Cobalt Connector [here](/integrations/integrationbuilder/#connection).
```

Comment:
Should we remove this step based on https://cobalthq.slack.com/archives/C05EYKJ2JKX/p1727104986073709?thread_ts=1727101885.091429&cid=C05EYKJ2JKX?
```suggestion
```